en/starter/settingup.md

```diff
@@ -24,7 +24,7 @@ redirect_from: "/starter/setting up.html"
 <hr>
 ## Ubuntu Linux
 
-Kitura is tested on Ubuntu 14.04 LTS, Ubuntu 15.10, and Ubuntu 16.04 LTS.
+Kitura is tested on Ubuntu 14.04 LTS.
```

Comment:
If our developers are depending on Travis-CI to test on Linux, that is a *BIG* problem. That is potential for build breaks which are *VERY* bad. All developers should be testing on both macOS and Linux.

We should have some of our developers working on Ubuntu 14.04 and others on Ubuntu 16.04.

Additionally there is an issue actively being worked on that gives us Travis-CI n Ubuntu 16.04